app/views/events/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container-show">
+  <h1><%= @event.title %></h1>
+  <div class="event">
+    <div class="description">
+      <p><%= @event.description %></p>
+    </div>
+    <div class="info">
+      <p><i class="fas fa-user-astronaut"></i> <%= @event.creator.name %></p>
+      <p><i class="far fa-calendar-alt"></i> <%= @event.starting_date_time.strftime("%B %-d, %Y at %l:%M %p") %></p>
+      <p><i class="far fa-calendar-alt"></i> <%= @event.ending_date_time.strftime("%B %-d, %Y at %l:%M %p") %></p>
+      <p><i class="fas fa-map-marked-alt"></i> <%= @event.location %></p>
+    </div>
+
+  </div>
+  <% if @event.attendees.include?(current_user) %>
```

Comment:
OPTIONAL

- [ ] It is a good practice to have zero logic in the views directory. Create methods in the helpers directory and call the methods in the views directory instead.
Read this [article](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4) for a head-start. Another great piece to read is this [article](https://brandonhilkert.com/blog/refactoring-logic-from-a-rails-view/)